cumulusci.yml

```diff
@@ -73,6 +73,13 @@ tasks:
         class_path: cumulusci.tasks.salesforce.Deploy
         options:
             path: unpackaged/config/delete
+    
+    ensure_record_type:
```

Comment:
The name and description here will be the default text shown for this step in the installer, so let's make sure it's user-friendly. Something like:

    name: Ensure Opportunity Record Types
    description: This will make sure record types are enabled for the Opportunity object in your org before installing NPSP. If there are no Opportunity record types yet, it will create one called NPSP Default.

(Check with docs/CCE for preferred language.)

And may as well set `sobject: Opportunity` here too, since that's the one we care about for NPSP.